kubernetes/resource_kubernetes_replication_controller_migrate.go

```diff
@@ -10,7 +10,7 @@ import (
 )
 
 func resourceKubernetesReplicationControllerV0() *schema.Resource {
-	schemaV1 := resourceKubernetesReplicationControllerSchemaV1()
+	schemaV1 := resourceKubernetesReplicationControllerV1SchemaV1()
```

Comment:
It looks a bit redundant. Could you please change `resourceKubernetesReplicationControllerV1SchemaV1` to `resourceKubernetesReplicationControllerV1Schema`(strip `V1` at the end)?